core/orchestrator.go

```diff
@@ -803,6 +811,14 @@ func (rtm *RemoteTranscoderManager) completeTranscoders(trans *RemoteTranscoder)
 	sort.Sort(byLoadFactor(rtm.remoteTranscoders))
 }
 
+func (rtm *RemoteTranscoderManager) totalCapacity() int {
```

Comment:
Maybe add a comment that the caller of this function needs to hold the RTmutex

Reply:
done